test/test.js

```diff
@@ -6,7 +7,7 @@ var eq = require('./shared/eq');
 
 describe('test', function() {
   it('returns true if string matches dotAll pattern', function() {
-    eq(R.test(/x.*z/s, 'x.\nyz'), true);  /* eslint-env es2018 */
+    eq(R.test(/x.*z/s, 'x.\nyz'), true);
   });
 
   it('returns true if string matches dotAll pattern', function() {
```

Comment:
This test case is duplicated with the upper one?

Reply:
I remove it.
but  `/* eslint-env es2018 */`comment  is Invalid
```
10:16  error  Parsing error: Invalid regular expression flag

✖ 1 problem (1 error, 0 warnings)

```

Reply:
I find out that, there is no eslint environment called `es2018`, the latest eslint only has es6, es2017, es2020 and es2021.

You can update `eslint` and `eslint-plugin-import` to the latest version, and replace `/* eslint-env 2018 */` to `/* eslint-env 2020 */`, and change some .eslintrc configs related to `eslint-plugin-import`.

But we can also just remove the `/* eslint-env 2018 */`, merge this PR, and update eslint in another pr later.

Reply:
`eslint-env` [Can't set env with a comment](https://github.com/eslint/eslint/issues/2705).
I remove the eslint-env comment. You can help me update eslint in another pr later.

Reply:
ok

Reply:
Can you disable eslint check in the line `eq(R.test(/x.*z/s, 'x.\nyz'), true);` temporarily ?

I will update eslint related things later.